wpengine/faustjs
# Merl 1749 faust should warn if the secret key is invalid

## Tasks


## Description



## Related Issue(s):



## Testing

To test this change, you can:

1. Switch to the PR branch
2. Run `npm i` at the root 
3. Run `npm run build` at the root
4. `cd` into the `examples/next/faustwp-getting-started` project
5. Check to ensure your env FAUST_SECRET_KEY and your key in your test site's Faust Headless Settings (wp-admin setting) both match.
-- <img width="500" alt="image" src="https://github.com/wpengine/faustjs/assets/50935135/da8fa4a1-e0a1-479c-a8db-38045be7075c">

6. Run `npm run dev` and ensure there is NO warning in the terminal window that says: `Check to ensure your FAUST_SECRET_KEY matches your Faust Secret Key under wp-admin settings`
7. Change your env FAUST_SECRET_KEY slightly so that it no longer matches the test site's Faust Headless Settings secret key.
-- <img width="500" alt="image" src="https://github.com/wpengine/faustjs/assets/50935135/98ea3a7c-9019-45de-9db8-15dc9f48ae18">

8. Run `npm run dev` and ensure there IS a warning in the terminal window:
<img width="898" alt="image" src="https://github.com/wpengine/faustjs/assets/50935135/bb2d9d92-0076-4334-bab4-92371386f1cb">

## Screenshots


## Documentation Changes



## Dependant PRs
